### PR TITLE
Search events

### PR DESCRIPTION
**Pull request by tiago-lp**
**Feature/Bug description:** Creating a page to search for events.

**Solution:** 
Usage examples:
Peek 2019-03-14 15-40
Peek 2019-03-14 15-41222
Peek 2019-03-14 15-413333

**TODO/FIXME:** n/a

### CODE REVIEW

File: frontend/search/search_event.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
+<white-toolbar title="PESQUISAR" primary-button-icon="close"></white-toolbar>
+
+<div style="overflow: scroll;">
+    <div class="search-mobile-options-title search-event-header" id="search-title" layout="row" layout-align="space-between">
+        <b>Pesquisa avançada</b>
+        <md-icon md-colors="{color: 'light-green-500'}" ng-if="!searchCtrl.hasNotSearched"
+            ng-click="searchCtrl.hasNotSearched = true; searchCtrl.clearFilters(); searchCtrl.refreshPreviousKeyword(); searchCtrl.search_keyword = ''">
```
Comment (MaianaB) [inline frontend/search/search_event.html]: Is not better create the function?

File: frontend/search/search_event.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+            arrow_drop_down
+        </md-icon>
+    </div>
+    <div ng-if="searchCtrl.hasNotSearched" layout="column" layout-align="center center" class="search-mobile-options" style="overflow: scroll">
```
Comment (MaianaB) [inline frontend/search/search_event.html]: move to ccs file

**Review by MaianaB — CHANGES_REQUESTED** [review]

**Review by MaianaB — APPROVED** [review]

File: frontend/search/search.css [hunk 3]
```diff
@@ -91,6 +91,48 @@
...
+}
+
+.search-event-input-name {
+  width: 90%;
+  margin-left: 1.2em;
+  font-size: 0.8em;
+}
+
+.search-event-input {
+  border-color: #009688 !important;
```
Comment (Luiz-FS) [inline frontend/search/search.css:105]: Is really necessary?

File: frontend/search/searchEventController.js [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+         * @param {Event} ev : The event that is useful to deal with the mdDialog.
+         * When the user isn't in a mobile its value is undefined. 
+         */
+        searchCtrl.search = function search() {
+            if (searchCtrl.search_keyword) {
+                let promise = searchCtrl.makeSearch(searchCtrl.search_keyword, 'event');
+                promise.then(() => {
+                    searchCtrl.setupResultsInMobile();
+                });
+                refreshPreviousKeyword();
```
Comment (Luiz-FS) [inline frontend/search/searchEventController.js]: Should not be within the promise?

**Review by Luiz-FS — CHANGES_REQUESTED** [review]

**Review by JuliePessoa — APPROVED** [review]

**Review by Luiz-FS — APPROVED** [review]

**Pull request merged**